client/payment-details/dispute-details/dispute-summary-row.tsx

```diff
@@ -0,0 +1,131 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import moment from 'moment';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { dateI18n } from '@wordpress/date';
+
+/**
+ * Internal dependencies
+ */
+import type { Dispute } from 'wcpay/types/disputes';
+import { HorizontalList } from 'wcpay/components/horizontal-list';
+import { formatCurrency } from 'wcpay/utils/currency';
+import { reasons } from 'wcpay/disputes/strings';
+import { formatStringValue } from 'wcpay/utils';
+import { isAwaitingResponse } from 'wcpay/disputes/utils';
+import { ClickTooltip } from 'wcpay/components/tooltip';
+import Paragraphs from 'wcpay/components/paragraphs';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+
+interface Props {
+	dispute: Dispute;
+	daysRemaining: number;
+}
+
+const DisputeSummaryRow: React.FC< Props > = ( { dispute, daysRemaining } ) => {
+	const respondByDate = dispute.evidence_details?.due_by
+		? dateI18n(
+				'M j, Y, g:ia',
+				moment( dispute.evidence_details?.due_by * 1000 ).toISOString()
+		  )
+		: '–';
+
+	const disputeReason = formatStringValue(
+		reasons[ dispute.reason ]?.display || dispute.reason
+	);
+	const disputeReasonSummary = reasons[ dispute.reason ]?.summary || [];
+
+	const columns = [
+		{
+			title: __( 'Dispute Amount', 'woocommerce-payments' ),
+			content: formatCurrency( dispute.amount, dispute.currency ),
+		},
+		{
+			title: __( 'Disputed On', 'woocommerce-payments' ),
+			content: dispute.created
+				? dateI18n(
+						'M j, Y, g:ia',
+						moment( dispute.created * 1000 ).toISOString()
+				  )
+				: '–',
+		},
+		{
+			title: __( 'Reason', 'woocommerce-payments' ),
+			content: (
+				<>
+					{ disputeReason }
+					{ disputeReasonSummary.length > 0 && (
+						<ClickTooltip
+							buttonIcon={ <HelpOutlineIcon /> }
```

Comment:
This icon doesn't align correctly. The way it is rendered inside `ClickTooltip` looks like changing is going to be complicated.

Mockup vs implemented.
![Screenshot 2023-09-01 at 4 12 13 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/06f99693-1ba9-49b0-9e37-63531f33bed7)
![Screenshot 2023-09-01 at 4 12 18 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/d9d4b64f-c6a5-434d-9d0e-ed7e4a38c195)

Reply:
I've added a suggestion that may solve this https://github.com/Automattic/woocommerce-payments/pull/7077#discussion_r1315233606

Reply:
#7124 has been merged, which should resolve this alignment issue.